### PR TITLE
Brain immunity/reformat files into separate pages

### PR DESCRIPTION
**Pull request by wongstein**
Split the Immunity docs into separate feature-focused pages for clarity and simplicity

### CODE REVIEW

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 1]
```diff
@@ -69,6 +69,18 @@ The release-scripts given are:
...
+2. **KONG_HOST**: The host name that Kong Admin API can be reached at.  If deploying with Docker Compose, this is going to be the name of the Kong container specified in the compose file.  If Kong Admin has been exposed behind a web address, KONG_HOST will be that web address.
+3. **KONG_PORT**: The port where Kong Admin can be found.  Together with KONG_PROTOCOL and KONG_HOST, Collector App is able to communicate with Kong Admin
+4. **KONG_ADMIN_TOKEN**: The authentication token used to validate requests for Kong Admin API, if configured.
+5. **SQLALCHEMY_DATABASE_URI**: The sql alchemy formated URI that points to the Postgres Database that collector App is using as a backend.  The format is: postgresql://<USER>:<PASSWORD>@<POSTGRES-HOST>:<POSTGRES-PORT>/collector
+6. **SLACK_WEBHOOK_URL**: The url of the slack channel that Immunity alert notifications should be sent to.  This url will be the default channel alerts are sent to, however more channels and rules configuring which alerts to send to which channel can be made.  See the [Adding a Slack Configuration](alerts.md#adding-a-slack-configuration)
```
Comment (lena-larionova) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]:
```suggestion
6. **SLACK_WEBHOOK_URL**: The url of the slack channel that Immunity alert notifications should be sent to.  This url will be the default channel alerts are sent to, however more channels and rules configuring which alerts to send to which channel can be made.  See the topic on [Adding a Slack Configuration](/enterprise/{{page.kong_version}}/brain-immunity/alerts#alert-slack-integration).
```

File: app/enterprise/1.3-x/brain-immunity/install-configure.md [hunk 2]
```diff
@@ -188,539 +200,9 @@ Requests to the status endpoint will confirm the Collector App is up and running
...
-curl -X POST http://<COLLECTOR_HOST>:<COLLECTOR_PORT>/trainer/reset
-```
-
-#### Configuring Auto-Training Rules
-
-For best use, Immunity retrains on a regular basis. If you do not feel like you need to retrain your models regularly and are happy with the current model you have now, you can stop auto retraining via post request to the /trainer/config endpoint. This endpoint takes these parameters:
+## Everything is ready to go
+Collector App and Kong are up and ready to analyze incoming traffic for [alerts](/enterprise/{{page.kong_version}}/brain-immunity/alerts), [auto-generate specs](/enterprise/{{page.kong_version}}/brain-immunity/auto-generated-specs), and display your traffic visually in [Service-Map](/enterprise/{{page.kong_version}}/brain-immunity/service-map).
 
+For any problems encountered while setting up Collector App, Collector Plugin, or configuring other aspects of Brain and Immunity, please check out [troubleshooting](troubleshooting.md) for help debugging common problems.
```
Comment (lena-larionova) [inline app/enterprise/1.3-x/brain-immunity/install-configure.md]: Looks like troubleshooting.md isn't exposed in the nav, need to add it. The link will also need to be: /enterprise/{{page.kong_version}}/brain-immunity/troubleshooting

File: app/_data/docs_nav_ee_1.3-x.yml [hunk 3]
```diff
@@ -281,6 +281,10 @@
   items:
     - text: Installation and Configuration
       url: /brain-immunity/install-configure
+    - text: Alerts
+      url: /brain-immunity/alerts
+    - text: Auto-Generated API Documentation
+      url: /brain-immunity/auto-generated-specs
     - text: Service Map
       url: /brain-immunity/service-map
```
Comment (lena-larionova) [inline app/_data/docs_nav_ee_1.3-x.yml]: Add an entry for Troubleshooting

**Comment by wongstein** [comment]
@lena-larionova Ready for re-review!!!

**Pull request merged**